### PR TITLE
Pgwar-statements

### PR DESCRIPTION
**Pull request by joschne**
Add pgwar.statement as described here: https://kleiolab.atlassian.net/wiki/spaces/ORG/pages/3730374663/Warehouse+statements

### CODE REVIEW

File: database/test/units/functions/pgwar-get-value-label-cell.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        string_value,
+        numeric_value,
+        fk_class
+    )
+VALUES(44, 55, 0, 1, 'foo', 43.21, 88);
+
+SELECT is(
+        pgwar.get_value_label(cell),
+        'foo',
+        'Assert tables.cell is correctly converted a label'
```
Comment (dferhod) [inline database/test/units/functions/pgwar-get-value-label-cell.sql:21]: Typo?
Assert tables.cell is correctly converted *to* (or *into*?) a label

File: database/test/units/functions/pgwar-get-value-label-dimension.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+INSERT INTO information.dimension (numeric_value, fk_measurement_unit, fk_class)
+SELECT 43.21,
+    m_unit.pk_entity,
+    123
+FROM m_unit;
+
+SELECT is(
+        pgwar.get_value_label(dimension),
+        '43.21',
+        'Assert information.dimension is correctly converted a label'
```
Comment (dferhod) [inline database/test/units/functions/pgwar-get-value-label-dimension.sql:20]: Typo?
Assert information.dimension is correctly converted to (or into?) a label

File: database/test/units/functions/pgwar-get-value-label-language.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+SELECT plan(1);
+
+INSERT INTO information.language (notes, iso6391, iso6392b, iso6392t, pk_language, fk_class)
+VALUES ('english', 'e', 'en', 'eng', 'eng', 123);
+
+SELECT is(
+        pgwar.get_value_label(language),
+       'english',
+        'Assert information.language is correctly converted a label'
```
Comment (dferhod) [inline database/test/units/functions/pgwar-get-value-label-language.sql:13]: same

File: database/test/units/functions/pgwar-get-value-label-place.sql [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+INSERT INTO information.place (geo_point, fk_class)
+VALUES (
+        ST_SetSRID(ST_MakePoint(7.123, 3.123), 4326)::geography,
+        123
+    );
+
+SELECT is(
+        pgwar.get_value_label(place),
+        'WGS84: 7.123°, 3.123°',
+        'Assert information.place is correctly converted a label'
```
Comment (dferhod) [inline database/test/units/functions/pgwar-get-value-label-place.sql:16]: same

**Review by dferhod — COMMENTED** [review]
See comments (nothing special, it's just a repeated typo in the unit test printed strings.

File: database/test/units/functions/pgwar-get-value-label-cell.sql (hunk 1 above)
Comment (joschne) [reply database/test/units/functions/pgwar-get-value-label-cell.sql:21]: yes, done

File: database/test/units/functions/pgwar-get-value-label-dimension.sql (hunk 2 above)
Comment (joschne) [reply database/test/units/functions/pgwar-get-value-label-dimension.sql:20]: yes, done

File: database/test/units/functions/pgwar-get-value-label-language.sql (hunk 3 above)
Comment (joschne) [reply database/test/units/functions/pgwar-get-value-label-language.sql:13]: yes, done

File: database/test/units/functions/pgwar-get-value-label-place.sql (hunk 4 above)
Comment (joschne) [reply database/test/units/functions/pgwar-get-value-label-place.sql:16]: yes, done

**Review by dferhod — APPROVED** [review]

**Pull request merged**